LayoutTests/fast/snapshot/positioned-static.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+#box {
+    width: 100px;
+    height: 100px;
+    background-color: blue;
+    color: black;
+}
+</style>
+</head>
+<body>
+    <div id="box">Is this included?</div>
+    <canvas id="canvas"></canvas>
+<script>
+if (window.testRunner) {
+    testRunner.waitUntilDone();
+}
+
+async function main() {
+    const box = document.getElementById('box');
+    const canvas = document.getElementById('canvas');
+
+    if (!window.internals) {
+        console.log('FAIL: window.internals is not available');
+        return;
+    }
+
+    const imageData = internals.snapshotNode(box);
+
+    if (!imageData) {
+        console.log('FAIL: snapshotNode returned null');
+        return;
+    }
+
+    canvas.width = imageData.width;
+    canvas.height = imageData.height;
+    
+    const ctx = canvas.getContext('2d');
+    ctx.putImageData(imageData, 0, 0);
+
+    box.remove();
+
+    if (window.testRunner) {
+        testRunner.notifyDone();
+    }
+}
+
+window.addEventListener('load', () => {
+    setTimeout(main, 200);
```

Comment:
Don't use a 200ms timeout. If every one of our thousands of layout tests took 200ms, they would take hours to run.